lessons/specifics/plug.md

```diff
@@ -83,10 +100,13 @@ end
 
 This supervises Cowboy, and in turn, supervises our `HelloWorldPlug`.
 
-Now, the `application` portion of `mix.exs` needs two things:
+In the `Plug.Adapters.Cowboy.child_spec/4` call, the third argument will be passed to `Example.HelloWorldPlug.init/1`.
+
+We're not finished yet. Open `mix.exs` again, and find the `applications` function.
```

Comment:
Are most people using 1.4 these days?  If so then mentioning `applications` could be confusing, we might want to update these lessons with the 1.4 changes.  That can be a separate PR though.